test/makeProcessors.js

```diff
@@ -124,4 +124,11 @@ describe('processors', () => {
       });
     });
   });
+  
+  describe('apollo-link-state', () => {
+    it('skips queries that contain the @client directive', () => {
```

Comment:
This test will currently always pass. The `KnownDirectives` validator needs to be added to[ the baseConfig.rules object](https://github.com/stipsan/eslint-plugin-graphql/blob/2bd3721a2b68c0444d8efa219e070ec1ef526399/test/makeProcessors.js#L14):

```js
baseConfig: {
  rules: {
    'graphql/required-fields': [
      'error',
      {
        schemaJson,
        env: 'literal',
        requiredFields: ['id'],
        validators: ['KnownDirectives'],
      },
    ],
  },
},
```